README.md

```diff
@@ -547,6 +547,15 @@ To completely delete the infrastructure:
 ansible-playbook -i inventory.yml -u root main.yml --tags purge
 ```
 
+### Outputs
+
+Based on your inventory, the complete state of the infrastructure (TF files, TF states, LUKS keys etc...)
+will be outputted to `${INVENTORY_NAME}-state.tar.gz` this file is important to track the
```

Comment:
better use "recorded" instead of "outputted"? What do you mean?

Reply:
Yep seems better, Fixed